src/BiddingFacility.cpp

```diff
@@ -1,57 +1,66 @@
 #include <iostream>
 #include "BiddingFacility.h"
 
+// Constructor
+
 BiddingFacility::BiddingFacility() {
     BiddingFacility::bid = new int(0);
 }
 
+// Destructor
+
+BiddingFacility::~BiddingFacility() {
+    delete bid;
+}
+
+// Gameplay Methods
+
+// TODO: Add verification to ensure that players can afford opening bid (for A1 assume user perfect inputs)
+// Returns the index of the player who goes first (ie. to determine player order)
 int BiddingFacility::biddingPhase(std::vector<Player*> players, int numberOfPlayers) {
```

Comment:
Just to double check, here you want to be passed  a vector of pointers to Players? Or is it supposed to be a pointer to a vector of Players?

Reply:
Its a vector that consists of Player pointers (each that point to a unique player object).